tests/pos/scala2-macro-compat-1.scala

```diff
@@ -0,0 +1,26 @@
+
+
+object Macro1 {
```

Comment:
this doesn't work if i try to call lineNumber:
```scala
object Macro1Consume {
  val test = Macro1.lineNumber
}
```

```scala
-- Error: macros2.scala:4:13 -------------------------------------
4 |  val test = Macro1.lineNumber
  |             ^
  |       cannot merge
  |         method lineNumber in object Macro1 of type => Int  and
  |         method lineNumber in object Macro1 of type => Int
  |       they are both defined in object Macro1 but have matching signatures
  |         => Int and
  |         => Int
  |       as members of object Macro1
  |              
1 warning found
1 error found
```

Reply:
It is already fixed. Probably by 71a20e4d32598478aaa5b338af6d4838c99d8bc8.